cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+timeout: 1200s
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20190906-745fed4'
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
you might want to enable this via the makefile instead

Reply:
Yep, I'll do that in a few minutes!